src/Analyzers/CSharp/CodeFixes/UseCollectionExpression/CSharpUseCollectionExpressionForArrayCodeFixProvider.cs

```diff
@@ -111,11 +105,11 @@ static bool IsOnSingleLine(SourceText sourceText, SyntaxNode node)
             {
                 ImplicitArrayCreationExpressionSyntax arrayCreation
                     => CSharpUseCollectionExpressionForArrayDiagnosticAnalyzer.TryGetMatches(
-                        semanticModel, arrayCreation, expressionType, cancellationToken),
+                        semanticModel, arrayCreation, expressionType, allowInterfaceConversion: true, cancellationToken, out _),
```

Comment:
during the fix portion, this value isn't relevant.  we're already only being called on cases we already analyzed.